### ISSUE TITLE
"ERROR: Only the default layer can be writen in UFO 2." for UFOs converted by tx

### ISSUE THREAD

**Issue by derwind** [adobe-type-tools/psautohint#229]
When I apply `psautohint` to UFO 2, it says "ERROR: Only the default layer can be writen in UFO 2." (to be precise, via `fontTools.ufoLib`)
I usually generate UFO 2 by `tx -ufo` or scripts as below:
```
python -c 'import defcon;f=defcon.Font("font.ufo");f.save("font2.ufo",formatVersion=2)'
```

If `psautohint` is used with -w option, it generates hinted glyphs.

Old `autohint` in AFDKO 2.5 build 65322 doesn't have this restriction and generates hinted glyphs under `glyphs.com.adobe.type.processedGlyphs`.

My AFDKO and `psautohint` versions are:
* AFDKO 3.2.0
* psautohint 2.0.0

I used Source Sans Pro and converted this UFO by `tx -ufo` for this test:
* https://github.com/adobe-fonts/source-sans-pro/tree/master/Roman/Instances/Regular/font.ufo

**Comment by miguelsousa** [comment]
> apply `psautohint` to UFO 2, it says "ERROR: Only the default layer can be writen in UFO 2."

This is working as designed. The UFO spec only started supporting layers in format 3. We developed a hybrid in-between format which `autohint` (and other AFDKO tools) supports, but while developing `psautohint` we made the decision to not propagate that hack any longer. Your options are to either convert the UFOs to format 3, or use `autohint` from an older AFDKO version.
